apps/emqx_retainer/src/emqx_retainer_api.erl

```diff
@@ -25,7 +25,7 @@
 -export([api_spec/0, paths/0, schema/1, namespace/0, fields/1]).
 
 -export([
-    lookup_retained_warp/2,
+    '/messages'/2,
```

Comment:
do you really want to use this fun name?

Reply:
Yes, many APIs use this name style